demo/README.md

```diff
@@ -1,8 +0,0 @@
-
-# Demo
-
-- `npm install --dev`
-- `npm run build`
-- open `index.html` in the browser of your choice
-
-
```

Comment:
undelete this pls

Reply:
Covered in c28bd847317f9b5c7eb0240b57da6392671144a7